src/main/java/core/basesyntax/dao/CarDao.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Car;
+import java.util.List;
+
+public interface CarDao {
+    void add(Car car);
+
+    Car getByModel(String model);
+
+    boolean remove(Car car);
+
+    void updateMaxSpeed(Car car, int maxSpeed);
```

Comment:
we can remove this as well

you can call `getByModel()`, get the car, and then change its maxSpeed